Makefile

```diff
@@ -1,3 +1,8 @@
+PWD=$(shell pwd)
+export C_INCLUDE_PATH=${PWD}/extern/tmp
+export LIBRARY_PATH=${PWD}/extern/tmp
+export LD_LIBRARY_PATH=${PWD}/extern/tmp
+
```

Comment:
So maybe using our own C_INCLUDE_PATH/LIBRARY_PATH/LD_LIBRARY_PATH instead of  system path is a good choice now?